src/resource_instances.py

```diff
@@ -15,18 +16,25 @@ def get_instances():
             resource_details = {'instance_id': resource['id'], 'name': resource['name'], 'region': resource['region_id'],
                                 'resource_group_id': resource['resource_group_id'], 'type': resource['type'],
                                 'resource_id': resource['resource_id'], 'state': resource['state'],
-                                'created_by': get_owner_details(resource['created_by']), 'created_at': resource['created_at'],
-                                'updated_at': resource['updated_at'], 'deleted_at': resource['deleted_at']}
+                                'created_by': '', 'created_at': resource['created_at'],
+                                'updated_at': resource['updated_at']}
             resource_instances.append(resource_details)
 
+            if(resource['type'] == "container_instance"):
```

Comment:
I'm not sure if this is true?
container_instance type belongs to cluster?

Reply:
I concur. I think it's possible to add it to the data spat out in `cluster_infrastructure.py`